### PR TITLE
Sunsetting Github AoR

### PR DESCRIPTION
**Pull request by malmstein**
Task/Issue URL: https://app.asana.com/0/1174433894299346/1204558888185157/f

### Description
This PR adds the changes described in the task above.
- New workflows to react to issues created
- New workflow for PR opened
- Updated Contributing guidelines to reflect we don’t take feature requests

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,9 +30,9 @@ After our team has investigated each issue, we will label them as:
 
 ## Contribute Code
 
-We welcome pull requests aimed at fixing bugs and security issues.
+We welcome Pull Requests aimed at fixing bugs and security issues. Refactoring, product changes and other features won't be considered and therefore the Pull Request will be closed.
 
-If you have a great idea or a feature request that can help improving our browser, start by opening a [Discussion](https://github.com/duckduckgo/Android/discussions). Discussions is the right place to have an open conversation about an idea with our devs and maintainers. Proposals that fit our product direction and timeline will be added to our backlog and labelled accordingly.
+If you have a great idea or a feature request, please refer to the Share Feedback session. Any issue that is created with the aim to request a feature will be closed. Proposals that fit our product direction and timeline will be added to our backlog and labelled accordingly.
```
Comment (aitorvs) [inline CONTRIBUTING.md:35]: What s a "Shared Feedback **session**"?

File: .github/workflows/ads-end-to-end.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Android Design System End-to-End tests
+ name: Android Design System End-to-End tests
```
Comment (aitorvs) [inline .github/workflows/ads-end-to-end.yml:1]: Why this extra indentation space?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ After our team has investigated each issue, we will label them as:
 
 ## Contribute Code
 
-We welcome pull requests aimed at fixing bugs and security issues.
+We welcome Pull Requests aimed at fixing bugs and security issues. Refactoring, product changes and other features won't be considered and therefore the Pull Request will be closed.
```
Comment (cmonfortep) [inline CONTRIBUTING.md:33]: Posted a comment about this bit https://app.asana.com/0/0/1204558888185157/1204888958021886/f

File: .github/workflows/action-pr-opened.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          action: 'add-asana-comment'
+
+      - name: Add Asana task to Internal Project / Incoming
+        uses: malmstein/github-asana-action@0.6.0
+        with:
+          asana-pat: ${{ secrets.GH_ASANA_SECRET }}
+          trigger-phrase: "Task/Issue URL:"
+          asana-project: '414730916066338'
+          asana-section: '1138897782858216'
+          action: 'add-task-asana-project'
```
Comment (aitorvs) [inline .github/workflows/action-pr-opened.yaml:42]: Why?

File: .github/workflows/action-pr-opened.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+      - name: Add Asana task to Android Releases Board Project / PR
+        uses: malmstein/github-asana-action@0.6.0
+        with:
+          asana-pat: ${{ secrets.GH_ASANA_SECRET }}
+          trigger-phrase: "Task/Issue URL:"
+          asana-project: '488551667048375'
+          asana-section: '488604899706669'
+          action: 'add-task-asana-project'
```
Comment (aitorvs) [inline .github/workflows/action-pr-opened.yaml:51]: Why?

File: CONTRIBUTING.md (hunk 1 above)
Comment (malmstein) [reply CONTRIBUTING.md:35]: typo, that should be a section.

File: .github/workflows/ads-end-to-end.yml (hunk 2 above)
Comment (malmstein) [reply .github/workflows/ads-end-to-end.yml:1]: typo, thanks for flagging.

File: .github/workflows/action-pr-opened.yaml (hunk 4 above)
Comment (malmstein) [reply .github/workflows/action-pr-opened.yaml]: removed

File: .github/workflows/action-pr-opened.yaml (hunk 5 above)
Comment (malmstein) [reply .github/workflows/action-pr-opened.yaml]: removed

**Comment by malmstein** [comment]
@cmonfortep can you please review this with the latest changes?

File: app/src/main/java/com/duckduckgo/app/survey/api/SurveyService.kt [hunk 6]
```diff
@@ -24,7 +24,7 @@ import retrofit2.http.GET
 @ContributesServiceApi(AppScope::class)
 interface SurveyService {
 
-    @GET("https://staticcdn.duckduckgo.com/survey/v2/survey-mobile.json")
+    @GET("https://ddg-sandbox.s3.amazonaws.com/survey/v2/survey-mobile.json")
```
Comment (cmonfortep) [inline app/src/main/java/com/duckduckgo/app/survey/api/SurveyService.kt:27]: Unrelated change. Seems a test endpoint for surveys.

File: .github/workflows/action-pr-opened.yaml [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+          action: 'create-asana-pr-task'
+
+      - name: Ensure Action sees template files
+        uses: actions/checkout@v2
+
+      - name: Render response template
+        id: render_template
+        uses: chuhlomin/render-template@v1.7
+        with:
+          template: .github/pr-reply-template.md
```
Comment (cmonfortep) [inline .github/workflows/action-pr-opened.yaml:56]: where is this file located `.github/pr-reply-template.md`?

File: .github/issue-pr-template.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
+Thank you for opening an Pull Request in our Repository.
```
Comment (cmonfortep) [inline .github/issue-pr-template.md:1]: This template is different from the one discussed in https://app.asana.com/0/1204853730139328/1204778116780347/f

Feedback:
- "The issue has been forwarded..."  -> That seems incorrect since we are in the PR template, probably copy & paste?
- "**is** the Pull Request is not" -> probably a typo? do you mean "**if** the Pull Request..."
- IMO, the whole part about contribution guidelines seems unnecessary and more related to the scenario of opening an issue, could we just use something similar to option 1 in https://app.asana.com/0/1204853730139328/1204778116780347/f

I don't have a strong choice here, so feel free to pick the one you prefer.

**Review by cmonfortep — APPROVED** [review]
Found 2 issues, please review them before merging.

LGTM 👍

File: app/src/main/java/com/duckduckgo/app/survey/api/SurveyService.kt (hunk 6 above)
Comment (malmstein) [reply app/src/main/java/com/duckduckgo/app/survey/api/SurveyService.kt]: good catch, removing

File: .github/workflows/action-pr-opened.yaml (hunk 7 above)
Comment (malmstein) [reply .github/workflows/action-pr-opened.yaml:56]: wrong name, updated
